src/main/features/response/views/partial-admission/how-much-have-you-paid.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layout.njk" %}
+
+{% set heading = 'How much money do you admit you owe?' %}
+
+{% from "form.njk" import saveAndContinueButton, csrfProtection, poundInput, errorSummary %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+
+    <form method="post" novalidate>
+
+      {{ csrfProtection(csrf) }}
+
+      <p>{{ t('The total amount claimed is {{ totalAmount }}. This includes the claim fee and any interest.',
+                { totalAmount: totalAmount | numeral }) }}</p>
+
+      {{ poundInput(label = '', name = 'amount', form = form) }}
```

Comment:
is this right? setting an empty label is smelly to me, is it accessible?

Reply:
This whole page was updated. Apparently the prototype was wrong.